examples/jsm/interactive/HTMLMesh.js

```diff
@@ -445,6 +445,28 @@ function html2canvas( element ) {
 
 			}
 
+			if ( element instanceof HTMLImageElement ) {
+
+				const drawImage = () => {
```

Comment:
I think it's better to define `drawImage()` outside of `drawElement()`. Otherwise the function gets defined over an over again.